napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -48,8 +52,38 @@
 )
 from napari.utils.translations import trans
 
+# TODO: add error icon and handle pip install errors
```

Comment:
Is there a proper issue opened in napari?

Reply:
I just added it to the list of items to address: 
https://github.com/napari/napari/issues/5396